wallet/src/de/schildbach/wallet/ui/main/WalletTransactionsFragment.kt

```diff
@@ -230,7 +230,7 @@ class WalletTransactionsFragment : Fragment(R.layout.wallet_transactions_fragmen
     }
 
     private fun showEmptyView() {
-        binding.walletTransactionsEmpty.isVisible = true
+        binding.walletTransactionsEmpty.isVisible = viewModel.transactionsLoaded
```

Comment:
Don't show "There are no transactions to display" if the app is still processing transactions to display.